### ISSUE TITLE
Update packages; use npm instead of yarn

### ISSUE THREAD

**Issue by holtwick** [scrumpy/tiptap#535]
Recently I'm running into more problems using TipTap. The problem is, that I am extending TipTap plugins and therefore importing some ProseMirror modules directly. Now the versions diverge and I get errors like "RangeError: Duplicate use of selection JSON ID cell" for tables.

So my proposal would be to either: 
1. Keep up to date with ProseMirror progress, or
2. Expose ProseMirror core features via TipTap

I also suggest switching to `npm`.

**Comment by holtwick** [comment]
Sorry, this isn't really a bug. I solve the problem with a fork for my special needs.

**Comment by philippkuehn** [comment]
Sorry, I've been a little inactive the last few weeks. What makes npm better than yarn in your opinion? I'm currently also bound to yarn because it supports workspaces.

**Comment by holtwick** [comment]
IMO npm is closer to the standard. I might be wrong, but I did want to solve my problems by using `npm link` to my fixed version, but that did not fully work as expected. However, I solved my issues by moving tiptap into my OnePile.app project now, because `Vue.extend()` in `ComponentView` did not work correctly when building everything separately. `yarn` is certainly a good choice as well. I have just been a little frustrated at the moment I wrote the ticket, because the simple upgrade of `prosemirror-table` crashed my top feature ;)
